openssl/src/nid.rs

```diff
@@ -104,7 +104,7 @@ pub enum Nid {
     G,
     S,
     I,
-    UID,
+    uid, // uniqueIdentifier
```

Comment:
Yeah, these variants should definitely be renamed - `G`, `S`, and `I` are not particularly reasonable to work with :). The `breaks` branch has the in-progress breaking changes that'll land in the next major release if you'd like to open a PR with the big rename against that.

In the mean time, let's add this as `UserId`.